control

```diff
@@ -1,7 +1,7 @@
 Package: com.haotestlabs.nfcactivator
 Name: NFCActivator
-Depends: mobilesubstrate, com.haotestlabs.nfcbackground, ws.hbang.common (>= 1.2), libactivator (>= 1.8.3)
-Version: 1.0.0-1
+Depends: mobilesubstrate, com.haotestlabs.nfcbackground | com.haotestlabs.nfcbackgroundenabler (>= 1.2.0.1), ws.hbang.common (>= 1.2), libactivator (>= 1.8.3)
```

Comment:
While NFCBackgroundEnabler isn't updated for this yet, it'll probably be alright.